apps/api/src/persistence/migrations/01/remove-base-digital-asset-url.migration.ts

```diff
@@ -0,0 +1,64 @@
+import { isNonEmptyString } from '@coscrad/validation-constraints';
+import { Photograph } from '../../../domain/models/photograph/entities/photograph.entity';
+import { Term } from '../../../domain/models/term/entities/term.entity';
+import { InternalError } from '../../../lib/errors/InternalError';
+import { DTO } from '../../../types/DTO';
+import { ArangoCollectionId } from '../../database/collection-references/ArangoCollectionId';
+import { DatabaseDocument } from '../../database/utilities/mapEntityDTOToDatabaseDTO';
+import { ICoscradMigration } from '../coscrad-migration.interface';
+import { ICoscradQueryRunner } from '../coscrad-query-runner.interface';
+
+type TermDocument = DatabaseDocument<DTO<Term>>;
+
+type PhotographDocument = DatabaseDocument<DTO<Photograph>>;
+
+/**
+ * This is the name of the environment variable where the base digital asset url
+ * (includes a trailing slash) should be stored.
+ */
+const BASE_DIGITAL_ASSET_URL = 'BASE_DIGITAL_ASSET_URL';
+
+export class RemoveBaseDigitalAssetUrl implements ICoscradMigration {
```

Comment:
Ok, I can see the pattern for migrations!

Reply:
Cool!